lib/db/mysql.js

```diff
@@ -674,13 +675,13 @@ module.exports = function (log, error) {
     )
   }
 
-  // Update : accounts
-  // Set    : emailVerified = true
-  // Where  : uid = $1
-  var VERIFY_EMAIL = 'CALL verifyEmail_3(?)'
+  // Update : accounts, emails
+  // Set    : emailVerified = true if email is in accounts table or isVerified = true if on email table
+  // Where  : uid = $1, emailCode = $2
+  var VERIFY_EMAIL = 'CALL verifyEmail_4(?, ?)'
```

Comment:
> I think it'd be worth changing that so that the auth-server passes in the emailCode in all cases.

Sounds good. Doing this will help simply some of the logic in the auth server.